app/models/StoredAttachment.scala

```diff
@@ -73,8 +73,8 @@ object StoredAttachment {
       operator    = attachment.operator,
       timestamp   = attachment.timestamp,
       url         = metadata.url,
-      fileName    = metadata.fileName,
-      mimeType    = metadata.mimeType,
+      fileName    = metadata.fileName.getOrElse("New Attachment"),
+      mimeType    = metadata.mimeType.getOrElse("FileType"),
```

Comment:
It might be better to do this in the places where the `fileName` and `mimeType` are used rather than here as this is dummy data for display purposes only, so perhaps it belongs in the Twirl templates where it is rendered?

Reply:
To explain this a bit more, we should not be able to download a file when we do not know the mime type.

In the binding-tariff-filestore, the [SuccessfulScanResult](https://github.com/hmrc/binding-tariff-filestore/blob/master/app/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResult.scala#L27) contains the [UploadDetails](https://github.com/hmrc/binding-tariff-filestore/blob/master/app/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResult.scala#L57). 

Upscan always returns this info, so this means that if our file was successfully virus scanned, we must know its name and mime type.

Reply:
This means that if we do not know the file's name or mime type, it must either have failed its virus scan or the scan has not completed yet